packages/manager/modules/pci/src/projects/project/project.routing.js

```diff
@@ -16,6 +16,17 @@ export default /* @ngInject */ ($stateProvider) => {
         format: 'json',
         value: ['.'],
       },
+      redirectTo: ($transitions) => {
```

Comment:
```suggestion
      redirectTo: /* @ngInject */ ($transitions) => $transitions.injector()
        .getAsync('project')
        .then((project) => project.status === 'creating'
          ? 'pci.projects.project.creating'
          : null
```

Reply:
Nop, redirectTo should not be injected. Please have a look in uiRouter documentation :)